chain/src/params.rs

```diff
@@ -41,6 +41,7 @@ impl From<AssetInfo> for pb::AssetInfo {
 pub struct ChainParams {
     pub chain_id: String,
     pub epoch_duration: u64,
+    pub unbonding_epochs: u64,
```

Comment:
`ChainParams` now contains an `unbonding_epochs` field, which will be accessible throughout the application.